code/modules/mob/abstract/new_player/character_traits.dm

```diff
@@ -71,4 +71,5 @@
 	H.disabilities |= ASTHMA
 	if(H.max_stamina)
 		H.max_stamina *= 0.8
-		H.stamina = H.max_stamina
\ No newline at end of file
+		H.stamina = H.max_stamina
+
```

Comment:
In future please try not to keep things in your commits if there are no meaningful changes to the file.

Reply:
I'm not sure why it touched that..